package.json

```diff
@@ -87,7 +87,8 @@
     "@rollup/pluginutils": "^4.2.1",
     "c8": "^7.12.0",
     "chai": "^4.3.6",
-    "eslint": "^8.23.1",
+    "eslint": "^8.23.0",
```

Comment:
Why have you bumped ESLint down?

Reply:
fixed. I had conflict there, and messed up the resolve!